src/Features/LanguageServer/Protocol/Handler/Completion/CompletionHandler.cs

```diff
@@ -215,6 +215,7 @@ bool IsValidTriggerCharacterForDocument(Document document, char triggerCharacter
                 Document document,
                 CompletionItem item,
                 CompletionResolveData? completionResolveData,
+                bool useVSCompletionItem,
```

Comment:
😕 This name doesn't give any indication that it's a "fake soft selection". I'm interested to see if there are any other alternatives.